swarm/storage/hasherstore.go

```diff
@@ -31,9 +31,10 @@ type chunkEncryption struct {
 
 type hasherStore struct {
 	store           ChunkStore
-	hashFunc        SwarmHasher
+	hashFunc        SwarmHash
```

Comment:
not sure i like this change. have you trued benchmarking with bmt?
thats already concurrent